controllers/subscription/subscription_controller.go

```diff
@@ -241,3 +259,33 @@ func (r *SubscriptionReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&operatorsv1alpha1.Subscription{}).
 		Complete(r)
 }
+
+func (r *SubscriptionReconciler) allowDatabaseUpdates(ctx context.Context, installation *integreatlyv1alpha1.RHMI, isServiceAffecting bool) error {
+	if installation.Status.ToVersion != "" && isServiceAffecting {
```

Comment:
@adam-cattermole , maybe it's possible to add log message, to see that we are in this function. It could help for RHOAM logs analyses. Thank you